mongodb/Acornfile

```diff
@@ -0,0 +1,331 @@
+args: {
+    // Specify whether MongoDB instance is a standalone or replica set
+    isReplicaSet: false
+    // Custom user to be created during the initialization
+    dbUserName: ""
+    // Custom database to be created during the initialization
+    dbName: "acorn"
+    replicas: 1
+    // Enable diagnostic mode
+    diagnosticMode: false
+    // Enable authentication
+    authEnabled: true
+    // Run arbiter node
+    // Mongodb additional command line flags
+    extraFlags: ""
+    arbiterEnabled: false
+    // Arbiter additional command line flags
+    arbiterExtraFlargs: ""
+    // Run hidden node
+    hiddenEnabled: false
+    hiddenReplicas: 1
+    // Hidden node additional command line flags
+    hiddenExtraFlargs: ""
+    // App installed namespace. NOTE(joseb): before fieldRef is supported,
+    appNamespace: "mongo"
+}
+profiles: {
+    prod: {
+        isReplicaSet: true
+        arbiterEnabled: true
+        hiddenEnabled: true
+        replicas: 3
+    }
+}
+    
+if !args.isReplicaSet {

```

Comment:
Is it possible to consolidate the two blocks (! isReplicaSet / isReplicaSet) into a single for loop, and condition off of 1 and >1?

Reply:
well, that is what I thought and tried at first, but finally added variable to specify architecture(replicaset/standalone) because one-node replicaset mongodb instance is different from standalone mongodb instance.